### PR TITLE
add avatar to user

### PR DESCRIPTION
**Pull request by t-romani**
user can add an avatar to its profile.
when no avatar is explicitly included, default one will be chosen.

### CODE REVIEW

**Review by gonzaM — APPROVED** [review]

**Comment by Alxzu** [comment]
I think either the description or the feature on this PR wasn't complete. If the feature is `Add avatar to a User` then you should be testing that, not just the empty case. Otherwise, the description of this PR is too short 😛

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -47,4 +47,20 @@
...
+
+    it 'is valid' do
+      expect(User.last.avatar.attached?).to eq(true)
+    end
+  end
+
+  context 'when image is nil' do
+    let!(:user) { create(:user, avatar: nil) }
+
+    it 'has default image' do
```
Comment (JuanfraM) [inline spec/models/user_spec.rb]: test should said `set a default image`

**Review by JuanfraM — COMMENTED** [review]
I think the avatar should be returned in the view of the user

File: app/views/api/v1/users/_info.json.jbuilder [hunk 2]
```diff
@@ -1,3 +1,4 @@
-json.id       user.id
-json.name     user.full_name
-json.email    user.email
+json.id        user.id
+json.name      user.full_name
+json.email     user.email
+json.avatar_id user.avatar.record.id
```
Comment (JuanfraM) [inline app/views/api/v1/users/_info.json.jbuilder]: I think you should use `polymorphic_url ` in order to return the path of the blob created

File: app/views/api/v1/users/_info.json.jbuilder [hunk 3]
```diff
@@ -1,3 +1,4 @@
-json.id       user.id
-json.name     user.full_name
-json.email    user.email
+json.id        user.id
+json.name      user.full_name
+json.email     user.email
+json.avatar polymorphic_url(user.avatar)
```
Comment (JuanfraM) [inline app/views/api/v1/users/_info.json.jbuilder]: align params here

**Review by JuanfraM — APPROVED** [review]
